ls1intum/Artemis
# Remove participation websocket subscription

### Checklist

### Motivation and Context
We still use too many websocket subscriptions. In this PR, we remove the subscription for `/topic/exercise/{exerciseId}/participation` because it is not really necessary.

### Description
When participating in an exercise, the invoked REST call already returns the changed participation. It does not really make sense to additionally send it via websocket, therefore we removed the websocket communication and now use the response and update the participation cache.

### Steps for Testing
1. Log in to Artemis
2. Navigate to Course Administration
3. Create file-upload/modeling/text exercises
4. Start the exercises
5. Navigate back and forth in the course overview / exercise details / submission page: check that the submission/result state is correct
6. Create a submission and submit
7. Navigate back and forth in the course overview / exercise details / submission page: check that the submission/result state is correct
